src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 [assembly: CLSCompliant(false)]
 [assembly: Guid("cdfd4e1b-bb5f-439b-8843-e6127cd9b8ad")]
 
-[assembly: AssemblyVersion("0.7.1")]
-[assembly: AssemblyFileVersion("0.7.1")]
+[assembly: AssemblyVersion("0.7.2")]
```

Comment:
@demyanenko you can revert the changes made to this file since we'll do the versioning later